pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -83,8 +83,43 @@ def build_xanes_map_api(start_id=None, end_id=None, *, param_file_name,
     used if properly generated XRF maps for the selected set of element emission line is
     already present in the .h5 files.
 
-
-    Format of the reference file (see parameter ``ref_file_name``).
+    Example of function call for the batch of scans with IDs in the range 92276-92335:
+
+    build_xanes_map(92276, 92335, param_file_name="param_335", scaler_name="sclr1_ch4",
+    sequence="load_and_process", ref_file_name="refs_Fe_P23.csv", emission_line="Fe_K",
+    emission_line_alignment="P_K")
+
+    Options:
+
+    Typical processing sequence includes operation of XRF map interpolation and stack alignment.
+    But those operations may be disabled by setting parameters ``interpolation_enable`` and
+    ``alignment enable`` to False. For example, if the position coordinates of XRF maps form
+    uniform grid, then disabling interpolation will reduce processing time.
+
+    There are two parameters that control stack alignment. The stack alignment for all emission
+    lines is performed based on the emission line selected for XANES (parameter ``emission_line``).
+    If maps for different emission line are better suited for alignment, then the default behavior
+    can be changed by setting the parameter ``emission_line_alignment``. The other parameter
+    controls the order of the stack alignment. The default order (starting from the top of
+    the stack or XRF map measured with highest energy) may be changed by setting the parameter
+    ``alignment_starts_from="bottom"``. In this case the alignment will start from the map
+    acquired with the lowest incident energy.
+
+    Incident energy used for generation of XRF maps is automatically computed as the lowest of
+    the energy read from the respective .h data file or the energy which activates the
```

Comment:
"... lowest of the energies read ..."
"... or the energy of the emission line of interest ..."

Reply:
Thanks. Actually I found the error in this statement: it has to be the highest energy of the two. Fixed.